R/latex_dependencies.R

```diff
@@ -30,10 +31,12 @@ latex_dependencies <- function(x = list()) {
 # in the head of a document
 latex_dependencies_as_string <- function(dependencies) {
   lines <- sapply(dependencies, function(dep) {
+    extra_lines <- paste(dep$extra_lines, collapse = ",")
```

Comment:
Why collapsing by a comma? I feel it should be `\n` instead.

Reply:
Oops, typo :persevere: 

Fixed!